.github/workflows/build.yml

```
@@ -137,6 +138,39 @@ jobs:
+            - name: Login to Quay.io
+              uses: docker/login-action@v3
+              with:
+                  registry: ${{ env.REGISTRY }}
+                  username: ${{ secrets.REGISTRY_USER }}
+                  password: ${{ secrets.REGISTRY_PASSWORD }}
+
+            - name: Export Tekton tasks
+              run: make export-tasks
+
+            - name: Push Tekton Bundle
+              run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:69: Double quote to prevent globbing and word splitting

(shellcheck)